dev/bots/test.dart

```diff
@@ -171,6 +171,17 @@ const Map<String, List<String>> kWebTestFileKnownFailures = <String, List<String
     'test/cupertino/scaffold_test.dart',
     'test/rendering/platform_view_test.dart',
   ],
+  'skwasm': <String>[
+    // These tests are not compilable on the web due to dependencies on
+    // VM-specific functionality.
+    'test/services/message_codecs_vm_test.dart',
+    'test/examples/sector_layout_test.dart',
+
+    // These tests are broken and need to be fixed.
+    // TODO(jacksongardner): https://github.com/flutter/flutter/issues/71604
+    'test/material/text_field_test.dart',
+    'test/widgets/performance_overlay_test.dart',
```

Comment:
The basic difference here is that the HTML renderer silently does nothing when the performance overlay is enabled, and the canvaskit and skwasm renderers actually throw. I think it's fine to just skip it this way for now, since that's what we're doing with CanvasKit.